srcpkgs/tarsnap/template

```diff
@@ -1,24 +1,22 @@
 # Template file for 'tarsnap'
 pkgname=tarsnap
-version=1.0.39
-revision=2
+version=1.0.40
+revision=1
 build_style=gnu-configure
 configure_args="--sysconfdir=/etc/${pkgname}"
 makedepends="e2fsprogs-devel openssl-devel"
 depends="bzip2 xz"
 conf_files="/etc/tarsnap/tarsnap.conf"
-short_desc="An online encrypted backup service (client)"
+short_desc="Tarsnap online encrypted backup service (client)"
```

Comment:
Repeating the package name isn't necessary, is it?
```suggestion
short_desc="Online encrypted backup service (client)"
```

Reply:
I am supremely annoyed by xlint's insistance on not starting with an article.